Cubical/Algebra/CommRing/LocalRing.agda

```diff
@@ -0,0 +1,229 @@
+{-
+  Definition of local rings.
+-}
```

Comment:
Maybe a comment like: "Note that local rings are more intricate constructively. See "A Course in Constructive Algebra" by Mines, Richman & Ruitenberg, p. 96 ff, where the two characterizations in this file are used as a definition of local ring, but without the requirement that 1 is different from 0."

Reply:
Nice, thanks! I also added a link to [the nlab page](https://ncatlab.org/nlab/show/local+ring).